src/views/validated-designs/index.tsx

```diff
@@ -20,26 +59,54 @@ export default function ValidatedDesignsLandingView({
 	categoryGroups,
 }: ValidatedDesignsLandingProps) {
 	return (
-		<BaseLayout mobileMenuSlot={<MobileMenuLevelsGeneric />}>
+		<BaseLayout mobileMenuSlot={<MobileMenuLevelsGeneric />} className={s.root}>
 			<Head>
 				<meta name="robots" content="noindex, nofollow" />
 			</Head>
-			<h1>{'HashiCorp Validated Designs'}</h1>
-			<ul>
+			<LandingHero
+				heading="HashiCorp Validated Design"
+				isHvd={true}
+				className={s.hvdHero}
+			/>
+			<div className={s.categoryGroupsContainer}>
 				{categoryGroups.map((category: HvdCategoryGroup) => (
-					<li key={category.slug}>
-						<h2>{category.title}</h2>
-						<p>{category.description}</p>
-						<ul>
+					<Card
+						elevation="base"
+						key={category.slug}
+						className={s.categoryGroupContainer}
+					>
+						<div className={s.categoryGroupHeader}>
+							<IconTileLogo size="large" productSlug={category.product} />
+							<Heading level={2} size={isMobile ? 400 : 500} weight="bold">
+								{category.title}
+							</Heading>
+						</div>
+						<Text
+							asElement="p"
+							size={isMobile ? 200 : 300}
+							weight="medium"
+							className={s.categoryGroupDescription}
+						>
+							{category.description}
+						</Text>
+						<div className={s.categoryGuidesContainer}>
 							{category.guides.map((guide: HvdGuide) => (
```

Comment:
for a11y reasons we should wrap this in a `<ul>` with each guide being wrapped in a `<li>`.

Reply:
fixed!